docs/boshdirector.adoc

```diff
@@ -1,20 +1,32 @@
-= Installing Bosh Director
+= Configuring Ops Manager Director
```

Comment:
Understand this, however all our PCF docs talk about configuring Ops Manager Director, not BOSH.  I too wish they would have just called it BOSH, but it is baked into our PCF docs and while I think it is good to highlight it, I think consistency with our official PCF docs is important to avoid confusion when the students refer back to them.

Oh and I just realized this needs further correction - it isn't just "BOSH Director" we are configuring and installing, it is actually BOSH (of which BOSH Director is one component).